src/components/Text/variants.ts

```diff
@@ -60,26 +62,46 @@ export const variants: Record<TextVariant, TextProp> = {
       lineHeight: '1.5rem',
     },
   },
-  a: {
-    element: 'a',
+  logo: {
+    element: 'p',
+    styles: {
+      fontFamily: '"Baloo 2", cursive',
+      fontStyle: 'normal',
+      fontWeight: '600',
+      fontSize: '2rem',
+      letterSpacing: '0.1em',
+      textTransform: 'uppercase',
+      lineHeight: 'initial',
+    },
+  },
+  label: {
+    element: 'span',
     styles: {
       fontFamily: '"Source Sans Pro"',
       fontStyle: 'normal',
+      fontWeight: '400',
+      fontSize: '1rem',
+      lineHeight: '1.5rem',
+    },
+  },
+  bold: {
+    element: 'span',
+    styles: {
+      fontFamily: '"Baloo 2"',
+      fontStyle: 'normal',
       fontWeight: '700',
       fontSize: '1rem',
       lineHeight: '1.5rem',
     },
   },
-  logo: {
-    element: 'p',
+  footer: {
+    element: 'span',
     styles: {
-      fontFamily: '"Baloo 2", cursive',
+      fontFamily: '"Source Sans Pro"',
       fontStyle: 'normal',
-      fontWeight: '600',
-      fontSize: '2.0rem',
-      letterSpacing: '0.1em',
-      textTransform: 'uppercase',
-      lineHeight: 'initial',
+      fontWeight: '400',
+      fontSize: '1rem',
+      lineHeight: '3.5rem',
     },
   },
   link: {
```

Comment:
I really want to refactor all these Baloo 2 cursive weight 400 links but maybe in the next PR. 😁